### PR TITLE
Job to automatic generate preview of pull request 🤖

### PR DESCRIPTION
**Pull request by Marvin9**
To make process of pull request review easy, Here is an initiate to tackle it using github action and [surge](https://surge.sh/).

### CODE REVIEW

File: scripts/deploy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const domain = `http://ignitus-preview-${gen}.surge.sh`;
+
+childProcess.exec(`./node_modules/.bin/surge ./build ${domain} --token ${SURGE_TOKEN}`, (err) => {
+  if (err) {
+    botComment(GITHUB_TOKEN, issuNumber, `Sorry @${USER}, Deploy Error 😶.`, () => {
+      process.exit(1);
+    });
+  } else {
+    botComment(GITHUB_TOKEN, issuNumber, `Yey @${USER}\n\n Liver Preview is ready: ${domain}\n😎`);
```
Comment (divyanshu-rawat) [inline scripts/deploy.js:22]:
```suggestion
    botComment(GITHUB_TOKEN, issuNumber, `Yey @${USER}\n\n Liver Preview is ready: ${domain}\n 🚀`);
```

Comment (divyanshu-rawat) [reply scripts/deploy.js:22]: What do you think? if we replace 😎=> 🚀

**Comment by Marvin9** [comment]
I understand your concern @divyanshu-rawat but Don't worry It is not included in bundle thus no issue on production side.

File: .github/workflows/pr-preview.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  deploy_preview:
+    name: Deploy Preview of PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - run: npm install yaml @actions/github
+      - run: echo "::set-env name=worker::$( node ./scripts/verify.js ${{ github.event.comment.user.login }} "${{ github.event.comment.body }}" )"
+      - if: env.worker == 'true'
```
Comment (Marvin9) [inline .github/workflows/pr-preview.yml:16]: I also considered performance of job. So job will further execute only if ```verified``` ***user*** and ***command***.

**Pull request merged**